README.md

```diff
@@ -1,3 +1,9 @@
+```diff
+- This is a DevOps ephemeral branch for configuring the CI/CD in AWS. There are no new features or code. 
```

Comment:
Not sure I understand this... This pr is being merged into develop so this message isn't applicable. Anyway dont think we need different messages on different branches. And dont need diff branch for ci/cd.